react/week4/managing-state/src/index.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import * as serviceWorker from './serviceWorker';
+
+// Importing components
+import Context from './context';
+import Searchuser from './searchuser';
+import ListItems from './userListItems';
+
+// Context
+const Contextprovider = Context.Provider;
+
+function FetchArray(event){
```

Comment:
You should only uppercase Component names. Functions and variables should still be camel case: 
```suggestion
function fetchArray(event){
```

It's necessary for react that components are uppercased: https://reactjs.org/docs/jsx-in-depth.html#specifying-the-react-element-type